src/overlays/actors/ovl_En_Zot/z_en_zot.c

```diff
@@ -1345,7 +1345,7 @@ void EnZot_Update(Actor* thisx, PlayState* play) {
 }
 
 Gfx* func_80B99580(GraphicsContext* gfxCtx) {
-    Gfx* dList = GRAPH_ALLOC(gfxCtx, sizeof(Gfx) * 2);
+    Gfx* dList = GRAPH_ALLOC(gfxCtx, sizeof(Gfx));
```

Comment:
In some places, it goes from:
`GRAPH_ALLOC(gfxCtx, sizeof(Gfx) * 2);` -> `GRAPH_ALLOC(gfxCtx, sizeof(Gfx));`
and in others:
`GRAPH_ALLOC(gfxCtx, sizeof(Gfx) * 2);` -> `GRAPH_ALLOC(gfxCtx, 2 * sizeof(Gfx));`
How do you determine which is right? Do they both always match?

Reply:
You look at how many displaylists/mtx the rest of the function actually uses to see how much space it needs.

Reply:
Just to expand on what Elliptic said, since the GRAPH_ALLOC now automatically aligns sizes to 16 byte values, and `Gfx` is only 8 Bytes, every odd number of `Gfx`s will also match the next even number above it since they have the same alignment.
i.e 1 -> 2, 3 ->4 etc.